src/common/Time.ts

```diff
@@ -13,6 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * Defines HrTime.
```

Comment:
```suggestion
 * Defines High-Resolution Time.
```

It can be more intuitive if the acronym is explained in the comment.